libraries/tests/mbed/serial_complete/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include "mbed.h"
+#include "test_env.h"
+
+int main() {
+    Serial *pc = new Serial(USBTX, USBRX);
```

Comment:
No indeed no specific reason.
I can make it global - so I will send an updated PR for this